src/globals.ts

```diff
@@ -434,6 +434,7 @@ export const SupportedLocales = Object.freeze([
   { name: 'Nederlands', code: 'nl' },
   { name: 'Português', code: 'pt' },
   { name: 'Русский', code: 'ru' },
+  { name: 'slovenščina', code: 'sl' },
```

Comment:
Minor change here, just getting this capitalized.

```suggestion
  { name: 'Slovenščina', code: 'sl' },
```